### PR TITLE
#1168 Wrap ftp source connector metadata table type with final form

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1168 )
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/pages/PipelinePage/Connectors/CustomConnector/CustomFinalForm.js [hunk 1]
```diff
@@ -226,7 +226,7 @@ class CustomFinalForm extends React.Component {
     });
 
     updateGraph({ update });
-  }, 1000);
+  }, 500);
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/PipelinePage/Connectors/CustomConnector/CustomFinalForm.js]: Is there a particular reason that we're changing this from _1000_ to _500_

**Comment by wu87988622** [comment]
retry manager

File: ohara-manager/client/src/components/pages/PipelinePage/Connectors/CustomConnector/customConnectorUtils.js [hunk 2]
```diff
@@ -182,6 +182,22 @@ export const replaceKeys = obj => {
   }, {});
 };
 
+export const rebackKeys = obj => {
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/PipelinePage/Connectors/CustomConnector/customConnectorUtils.js]: Just a note here. We shouldn't create two similar functions that share most of the logic together. 

I will fix this in another PR as I've already had a function which can change the key name separator from `.` to `_` and vice versa :)

**Comment by konekoya** [comment]
@wu87988622 don't forget to put the issue number at the beginning of the title and description as well

**Comment by konekoya** [comment]
Build failed is not related to this PR

File: ohara-manager/client/src/components/pages/PipelinePage/Connectors/CustomConnector/CustomFinalForm.js (hunk 1 above)
Comment (konekoya) [reply ohara-manager/client/src/components/pages/PipelinePage/Connectors/CustomConnector/CustomFinalForm.js]: Alright, let's see if this works out fine

Comment (wu87988622) [reply ohara-manager/client/src/components/pages/PipelinePage/Connectors/CustomConnector/CustomFinalForm.js]: I feel to slow some time is not work

File: ohara-manager/client/src/components/pages/PipelinePage/Connectors/CustomConnector/customConnectorUtils.js (hunk 2 above)
Comment (wu87988622) [reply ohara-manager/client/src/components/pages/PipelinePage/Connectors/CustomConnector/customConnectorUtils.js]: ok

**Pull request merged**